forge-gui/res/cardsfolder/upcoming/ogre_battlecaster.txt

```diff
@@ -4,12 +4,14 @@ Types:Creature Ogre Shaman
 PT:3/3
 K:First Strike
 T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigTarget | TriggerDescription$ Whenever CARDNAME attacks, you may cast target instant or sorcery card from your graveyard by paying {R}{R} in addition to its other costs. If that spell would be put into a graveyard, exile it instead. When you cast that spell, CARDNAME gets +X/+0 until end of turn, where X is that spell's mana value.
-SVar:TrigTarget:DB$ Pump | ValidTgts$ Instant.YouOwn,Sorcery.YouOwn | TgtZone$ Graveyard | RememberObjects$ Targeted | SubAbility$ DTCast
+SVar:TrigTarget:DB$ Pump | ValidTgts$ Instant.YouOwn,Sorcery.YouOwn | TgtZone$ Graveyard | RememberObjects$ Targeted | TgtPrompt$ Choose an instant or sorcery spell | SubAbility$ DTCast
```

Comment:
Always try to match oracle closely: `TgtPrompt$ Select target instant or sorcery card`